src/lib/provable/gadgets/elliptic-curve.ts

```diff
@@ -68,7 +73,7 @@ function add(p1: Point, p2: Point, Curve: { modulus: bigint; a: bigint }) {
 
   assert(
     Curve.modulus > l2Mask + 1n,
-    'Base field moduli smaller than 2^176 are not supported'
+    `Base field moduli smaller than ${l2Mask + 1n} are not supported`
```

Comment:
can you revert this? a random-looking decimal string will be harder to read than `2^176`

Reply:
I agree about the readability.  Still think it's a shame to hard code a value like this that can go out of date, but I'm ok accepting readability as the dominant factor.